### DISCUSSION TITLE
Help to change the area of Scan

### DISCUSSION THREAD

**Discussion by markolino85** [journeyapps/zxing-android-embedded#615] [Q&A]
Hello,
i am an appasionate of Android programming.
I use this library and it's work fine, so in my code when i tap a button the scanner start :
```
IntentIntegrator integrator = new IntentIntegrator(this);
                integrator.setOrientationLocked(true);
                integrator.setPrompt("any strings");
                integrator.setCaptureActivity(ActivityCaptureCustom.class);
                integrator.initiateScan();
```
And of course i handle the result.
I've need to have the rectangle of scanner smaller, how to change it's  height and width ?
Is it possible?
Thanks a lot

**Comment by antonfcss** [comment]
I support this question. How to use the scanner not on the full screen?
